**Kadaknath**

Kadaknath, also known as Kali Masi, is a black-bone chicken breed native to the Jhabua and Dhar districts of Madhya Pradesh. Kadaknath has three distinctive phenotypes (i.e., Jetblack, Pencil, and Golden). Kadaknath has a geographical indication (GI Tag) tag approved by the government of India in 2018. The supplementary data and information provided in this project are part of the below manuscript.


**Kadaknath chicken specific hitchhiking of BPIL with the FM locus**

Sagar Sharad Shinde 1 , Ashutosh Sharma 1 , Nagarjun Vijay 1

1 Computational Evolutionary Genomics Lab, Department of Biological Sciences, IISER Bhopal,
Bhauri, Madhya Pradesh, India
*Correspondence: nagarjun@iiserb.ac.in

Data is organised into the following folders:

**Fig1: This folder contains the following subfolders:**

    Fig1A: This folder contains R code for the Asia and China map generation. A map of Asia and China used in Fig1 has been provided.

    Fig1B: This folder contains an input file, code and figure for 101 chicken individuals genomewide PCA analysis.

    Fig1C: This folder contains the input file, code and figure for 101 chicken individuals Admixture analysis at best K=7.

**Fig2:** This folder contains the input files, code and figures for figure 2 and its zoomed view for the base level junction of Dup1 and Dup2 in black-bone(GOLD) and non-black-bone chicken(RJF) individuals.

**Fig3: This folder contains the following subfolders:**

    Fig3A: This folder contains the input files, code, and figure for the pairwise Fst comparison between Kadaknath (KADK) and Chinese black-bone chicken (CHIN).

    Fig3B: This folder contains the input files, code, and figure for the pairwise Fst comparison between Kadaknath (KADK) and Chinese black bone chicken (CHIN) in 10Kb, Pi, and iHS of chromosome 20.
    
**Fig4:** This folder contains the input files, code, and figure for the genomewide pairwise Fst comparison in 50 Kb window between JETB, PENC and GOLD.

**Fig5: This folder contains the following subfolders:**

Fig5A: This folder contains the input files, code, and figures for the boxplot of ABBA-BABA between Kadaknath(KADK), Chinese black-bone(CHIN) and commercial(BROC/CWLH/WHLL/RIRL) chicken populations.

Fig5B: This folder contains the input files, code, and figures for the Fst, D-statistics, and the number of ABBA-BABA sites between Kadaknath(KADK), Chinese black-bone(CHIN) and commercial(WHLL)chicken populations for each chromosome in 50 Kb window.

**BPIL_gene_alignment:** This folder contains an amino acid multifasta file, alignment file and supplementary figure of Kadaknath-specific nonsynonymous changes in the BPIL gene.

**Boxplot_of_Pi_Theta_of_black_and_commercial_chicken:** This folder contains the input files, code, and figures for the boxplot of Pi and Theta, Tajima's D and Fu ald Li's D of black-bone, native and commercial chicken populations.

**Boxplot_of_Pi_Theta_within_Kadak_and_KADAK_vs_CHIN_YOSK:** This folder contains the input files, code, and figures for the boxplot of Pi and Theta of Kadaknath (KADK), Chinese black-bone (CHIN ) and Yeonsan Ogye (YOSK) populations. It also contains input files, code, and figures for the boxplot of Pi and Theta within Kadaknath.

**Chromosome_wise_Fst_Pi_Theta_and_Dxy_GOLD_PENC_JETB:** This folder contains the input files, code, and figures for the pairwise comparison of Fst, Pi, Theta, and Dxy between GOLD, PENC and JETB for each chromosome.

**Chromosome_wise_Fst_Pi_Theta_and_Dxy_Jhabua_Bhopal:**This folder contains the input files, code, and figures for the pairwise comparision of Fst, Pi, Theta, and Dxy between Jhabua and Bhopal for each chromosome.

**Chromosome_wise_repeat_density:** This folder contains input files, code, and figures for the chromosome-wise repeat density with Fst in 50Kb window in Galgal6 genome assembly repeats downloaded from UCSC genome browser.

**IBD_analysis:** This folder contains input files, code and figures of Isolation By Distance (IBD) analysis for four combinations of multiple black bone chicken populations.

**Jhabua_vs_Bhopal_FST:** This folder contains the input files, code, and figure for the genomewide pairwise Fst comparison in 50Kb window between BH1(JETB1, PENC1, and GOLD1), BH2(JETB2, PENC2, and GOLD2) and JHA(JETB3, PENC3, and GOLD3).

**MT_Network:** This folder contains input files, code, and figures of Mitochondrial network analysis using the median-joining method in PopART and SplitsTree of the whole MT genome from 33 black bone chicken populations.

**Pi_and_Theta_of_FM_region:** This folder contains the input file, code and figure for Pi and Theta estimation of KADK, CHIN, BROC, RIRL, CWLH, YOSK, and WHLL populations using ANGSD.

**Population_specific_private_allele_count:** This folder contains the input files, code and figures for private allele count with and without dbSNP count.

**SFS:** This folder contains input files, code and figure for Site Frequency Spectrum (SFS) estimation for within kadkanath morphs (GOLD,PENC,JETB) and between KADK, CHIN, YOSK,and Commercial(BROC/CWLH/WHLL/RIRL) populations.

**SNPhylo_tree_101_and_34_black_bone_individuals:** This folder contains the input files, code, and figures for constructing a genomewide phylogenetic tree for all 101 and only 34 black-bone individuals using SNPhylo, which uses VCF files from bcftools and freebayes.

**Supplementary_Fig_ABBA_BABA_boxplot:** This folder contains input,code and figures for supplementary boxplot of ABBA-BABA Pattern in 50 kb window using ABBABABAwindows.py method.

**Supplementary_fig_of_ABBA_BABA_pattern_between_black_bone_and_commercial_breeds: This folder contains the following subfolders:**

    Calculate_Dxy_blackbone_vs_commercial_breeds: This folder contains the input files, code, and figures for the estimation of Dxy between black-bone and commercial breeds.

    BROC: This folder contains the input files, code, and figures for the Fst, D-statistics, and the number of ABBA-BABA sites between Kadaknath(KADK), Chinese black-bone(CHIN) and commercial(BROC) chicken populations for each chromosome and three candidate region GABRA2, C4ORF33,PRMT7 in 50 Kb window.

    CWLH: This folder contains the input files, code, and figures for the Fst, D-statistics, and the number of ABBA-BABA sites between Kadaknath(KADK), Chinese black-bone(CHIN) and commercial(CWLH) chicken populations for each chromosome and three candidate region GABRA2, C4ORF33,PRMT7 in 50 Kb window.

    RIRL: This folder contains the input files, code, and figures for the Fst, D-statistics, and the number of ABBA-BABA sites between Kadaknath(KADK), Chinese black-bone(CHIN) and commercial(RIRL) chicken populations for each chromosome and three candidate region GABRA2, C4ORF33,PRMT7 in 50 Kb window.

    WHLL: This folder contains the input files, code, and figures for the Fst, D-statistics, and the number of ABBA-BABA sites between Kadaknath(KADK), Chinese black-bone(CHIN) and commercial(WHLL) chicken populations for each chromosome and three candidate region GABRA2, C4ORF33,PRMT7 in 50 Kb window.

**Supplementary_figure_Admixture: This folder contains the following subfolders:**

    101_Genomewide_with_best_K: This folder contains the input file, code and figure for 101 chicken individuals genomewide Admixture analysis at 2 to 10 K and best K=7.

    101_individual_exclude_Z_W_scaffold_MT: This folder contains input files, code and figures for 101 chicken individuals genomewide Admixture analysis after excluding sex chromosomes(Z and W) and scaffolds for 2 to 10 K .

    34_individual_exclude_Z_W_scaffold_MT: This folder contains input files, code and figures for 34 black bone chicken individuals genomewide Admixture analysis after excluding sex chromosomes (Z and W) and scaffolds for 2 to 10 K.

    34_individual_genome_wide: This folder contains input files, code and figures for 34 black bone chicken individuals genomewide Admixture analysis for 2 to 10 K.

    34_individuals_FM_region: This folder contains input files, code and figures for FM locus regions (Flank1,Dup1,Int,Dup2,and Flank2) Admixture analysis for 34 black bone chicken individuals for 2 to 10 K.
    
**Supplementary_fig_PCA: This folder contains the following subfolders:**

    101_individual_exclude_Z_W_scaffold_MT: This folder contains input files, code and figures for 101 chicken individuals genomewide PCA analysis after excluding sex chromosomes(Z and W) and scaffolds.

    101_individual_genomewide: This folder contains input files, code and figures for 101 chicken individuals genomewide PCA analysis.

    101_individuals_FM_region: This folder contains input files, code and figures for FM locus region PCA analysis for 101 chicken individuals.

    34_individual_exclude_Z_W_scaffold_MT: This folder contains input files, code and figures for 34 black bone chicken individuals genomewide PCA analysis after excluding sex chromosomes(Z and W) and scaffolds.

    34_individual_genomewide: This folder contains input files, code and figures for 34 black bone chicken individuals genomewide PCA analysis.

    34_individuals_FM_region: This folder contains input files, code and figures for FM locus region PCA analysis for 34 black bone chicken individuals.
    
**Supplementary_of_main_figure_2: This folder contains the following subfolders:**

    Finding_of_rearranged_junction_using_PacBio_read: This folder contains the input file, code and figure for the rearranged junction.

    FM_region_read_coverage: This folder contains the input file, code and figure for read coverage of FM locus.

    Fst_Dxy_Fm_region.zip: This folder contains the input file, code and figure for estimation of Fst and Dxy in the FM region.

    igv_screenshot_of_rearranged_junction_regions.zip: This folder contains IGV screenshots of rearranged junctions of the FM region.

    Normalized_Coverage_of_Dup1_and_Dup2_baselevel_start_end_junction: This folder contains the input file, code and figure for coverage of base level start and end of Dup1 and Dup2.

    Normalized_Coverage_plot_of_rearranged_junction_regions.zip: This folder contains the input file, code and figure for the coverage plot of base level start and end of Dup1 and Dup2.
    
**VCF_file_generation:** This folder contains the genomewide variant calling code, removing indels, extracting common SNPs and generating a VCF file.

**iHS_XP-EHH_Fst_comparision_Dxy_main_fig3_AB_supplementary: This folder contains the following subfolders:**

    dxy_vs_Fst_between_CHIN_KADK: This folder contains the input file, code and figure for Dxy and Fst comparison between Kadaknath (KADK) and Chinese black-bone (CHIN) individuals using popgenWindows.py.

    Fst_comparision_and_cor.test: This folder contains the input file, code and figure for Fst comparison from ANGSD and popgenWindow.py between Kadaknath (KADK) and Chinese black-bone (CHIN) individuals and the correlation between these two methods.

    iHS_XP-EHH: This folder contains the input file, code and figure for iHS and XP-EHH comparison between Kadaknath (KADK) and Chinese black-bone individuals (CHIN).
    
**phastCons_conservation_score:** This folder contains input files, code and figures of Pencil specific fixed site and conservation score of 1.5KB from phastCons.

**Prerequisites:**

Bwa 0.7.17-r1188
Samtools 1.9
SNPhylo 20180901
Freebayes v1.0.0
Figtree 1.4.2
SplitsTree 4.14.8
PopART
Angsd 0.935-56-g6418514
Bedtools v2.26.0
popgenWindows.py
ABBABABAwindows.py
VCFtools 0.1.15
Picardtools
NGSadmix
PCAngsd 0.99
CLUSTALW
bcftools 1.9
snpEff
bam-readcount

R
library(sp)
library(reshape)
library("ecodist")
library(qqman)
library(gtools)
jackKnife.R
library(rehh)
library(tidyverse)
